setup.py

```diff
@@ -28,16 +27,18 @@
     install_requires=[
         'pandas',
         'numpy',
+        'scipy',
```

Comment:
what do we need scipy for? it's a pretty big dependency..

Reply:
I wanted to remove the try/except from the beginning of the `functions.py` file. But I can recover it if you preffer.

Reply:
Yeah, I'm torn. Most people won't use bounded normal in their models, and it seems a bit silly to make everyone install it if they aren't going to use it *at all*. Another solution is that we could implement our own bounded normal. 

If we can remove the scipy dependency, I think i'd prefer not to have it.

Reply:
Yes I agree using scipy for only one function is not worthly.

Reply:
Hi @JamesPHoughton ,
About removing the try/except for the `bounded  normal` and use always the scipy version. I just created and enviroment with conda using the _requirements.txt_ and I found out that _scipy_ was installed, I supposed that this was an optional requirement of _xarray_ and I found it in [their documentation](http://xarray.pydata.org/en/stable/installing.html). It is neccesary for netCDF I/O (we are not using that), and for acceleration in interpolating xarrays (we are using that for external lookups and data).